scss/colors/synthesis-palette.scss

```
@@ -0,0 +1,36 @@
+// stoplight color system
+$synth-green-1-success: #015029;
+$synth-green-2-success: #09914f;
+$synth-green-3-success: #d4f6e5;
+
+$synth-red-1-error: #c71f1f;
+$synth-red-2-error: #e03131;
+$synth-red-3-error: #fae1e1;
+
+$synth-yellow-1-warn: #fbda36;
+$synth-yellow-2-warn: #ffe356;
+$synth-yellow-3-warn: #fff1ab;
+
+// neutrals
+$synth-alert-bg-gray: #f4f4f4;
+$synth-div-stroke-gray: #dedede;
+$synth-header-gray: #fbfbfb;
+$synth-page-white: #fdfdfd;
+$synth-text-black: #1b1b1b;
+$synth-unselected-gray: #5b5b5b;
+
+// main schema - visual hierarchy
+$synth-accent-green: #158d71;
+$synth-dark-background-selected-blue: #162c4e;
+$synth-hover-state: #eef9f6;
+$synth-indicator-stroke-green: #91cabb;
+$synth-navbar-blue: #010812;
+$synth-primary-cta-blue: #162c4e;
+$synth-selected-state-green: #c6e2db;
+
+// feedback and signifier colors
+$synth-hyperlink: #3f6dca;
```

```suggestion
$synth-hyperlink-color: #3f6dca;
```

Might want to be more specific?

---

sounds fine to me